Sources/KSCrashRecordingCore/KSMachineContext.c

```diff
@@ -33,6 +33,10 @@
 
 #include <mach/mach.h>
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 180000  // Xcode 16.0
```

Comment:
May want to match the upstream change with `__has_include`

https://github.com/kstenerud/KSCrash/blob/06b35a3cde97f23aa58b09a52ec0a986c017b751/Sources/KSCrashRecordingCore/KSMachineContext.c#L31C1-L33C7

Reply:
changing it